tests/OpenApi/Factory/OpenApiFactoryTest.php

```diff
@@ -332,6 +332,7 @@ public function testInvoke(): void
         $components = $openApi->getComponents();
         $this->assertInstanceOf(Model\Components::class, $components);
 
+        // TODO: $components is almost completely empty except securitySchemes whereas its not the case if dumping data inside the classes like SchemaFactory
```

Comment:
What's todo here?

Reply:
Tests don't pass because $openApi->getComponents() is almost empty for now; It may be due to a wrong initialization of $factory